Chap_Introduction.tex

```diff
@@ -8,24 +8,24 @@ \chapter{Introduction}
 \label{chap:intro}
 
 \added{ \ac{PMIx} is an application programming interface specification to provide
-High Performance Computing libraries and programming models with
-portable and well defined access to commonly needed
-distributed computing system software.
-The \ac{PMIx} community strives to define interfaces to support the most widely
-used HPC programming models and libraries.
-\ac{PMIx} gives distributed systems software providers a better understanding of what
-services can be provided to best support these programming models.
-It also encourages the implementation of \ac{PMIx} APIs which allow for
+libraries and programming models with a portable and well-defined access to commonly
+needed services in distributed and parallel computing systems.
+A typical example of such a service is the exchange of information to start a \ac{HPC} service
+or application in a portable and scalable manner.
+As such, \ac{PMIx} also gives distributed system software providers a better understanding of how
```

Comment:
remove typical
Don't feel we even need the "token" HPC reference.  Just say "start a parallel application or distributed service in a.."